packages/alert/package.json

```diff
@@ -22,11 +19,14 @@
     "react": "^16.8.0 || 17.x",
     "react-dom": "^16.8.0 || 17.x"
   },
-  "main": "dist/index.js",
-  "module": "dist/alert.esm.js",
-  "typings": "dist/index.d.ts",
+  "main": "dist/reach-alert.cjs.js",
```

Comment:
just checking: were u using `exports: 'named'` for CJS output or are you sticking to the named exports only?

Reply:
I believe I'm exposing everything as a named export and pretty sure I've updated the docs accordingly, but we had some default exports before that I've kept around for back compat. Is there something I need to be looking for that might break this?

Reply:
Ah sorry you meant specifically for CJS -- yes, I previously used `exports: "named"` in rollup.

Reply:
Cool, I just have wanted to make sure that using Preconstruct didn't change the exported CJS shape for your packages - which could happen for `exports: 'auto'` (default) and lone default exports. 👍